### PR TITLE
UI fixes

### LINKED ISSUE #335: Add links to multiQC reports
The current seqr pipeline generates a number of project-level QC reports that must be visually inspected during processing. These reports are not versioned such that the latest version of the report is accessible at a stable URL in the project web bucket.  Currently, there is no way to navigate to a given report from a given project, rather you must construct the URL you wish to view (which gets old fast).

It would be super convenient if we could add a section to the[ project page](https://sample-metadata.populationgenomics.org.au/project/acute-care) that contains links to each of the relevant qc report types. The specific reports I had in mind currently are:
- [WGS Cram MultiQC](https://main-web.populationgenomics.org.au/acute-care/qc/cram/multiqc.html): `https://main-web.populationgenomics.org.au/{DATASET}/qc/cram/multiqc.html`  
- [WGS GVCF MultiQC](https://main-web.populationgenomics.org.au/acute-care/qc/gvcf/multiqc.html): `https://main-web.populationgenomics.org.au/{DATASET}/qc/gvcf/multiqc.html`
- [WGS FASTQC MultiQC](https://main-web.populationgenomics.org.au/acute-care/qc/fastqc/multiqc.html): `https://main-web.populationgenomics.org.au/{DATASET}/qc/fastqc/multiqc.html`
-  [Exome Cram MultiQC](https://main-web.populationgenomics.org.au/acute-care/exome/qc/cram/multiqc.html): `https://main-web.populationgenomics.org.au/{DATASET}/exome/qc/cram/multiqc.html`  
- [Exome GVCF MultiQC](https://main-web.populationgenomics.org.au/acute-care/exome/qc/gvcf/multiqc.html): `https://main-web.populationgenomics.org.au/{DATASET}/exome/qc/gvcf/multiqc.html`
- [Exome FASTQC MultiQC](https://main-web.populationgenomics.org.au/acute-care/exome/qc/fastqc/multiqc.html): `https://main-web.populationgenomics.org.au/{DATASET}/exome/qc/fastqc/multiqc.html`

Some reports may not exist for some projects either because:
- they have not been run yet
- they do not have both exome and genome data types
- some reports (fastqc) are not run by default 
We could muck around and only show links for reports that exist, however, I assume that would be non-trivial to establish so I would rather a fast interim solution that presents the URLs for all reports for all (RD) projects.

The links could be added into the current summary stats table at the top of the page or given their own section.

While we are adding useful links - adding a link to the relevant seqr project page would also be an awesome addition.

### PR DESCRIPTION
**Pull request by danielreti**
- Code cleanup
- Fix multi sequence to one sample display (https://centrepopgen.slack.com/archives/C020PDNH3N0/p1668645940001289)
- Add neater handling of null values in sequence information
- Sort batch table with no-batch at the end
- Added MultiQC Report linkouts (Fixes #335) 
- Replaced groupby (itertools) with group_by (utils)
- Display files in table instead of stringify

### CODE REVIEW

**Review by illusional — APPROVED** [review]
Minor feedback to address, but looks good!

File: web/src/DetailedInfoPage.tsx [hunk 1]
```diff
@@ -230,10 +237,20 @@ export const DetailedInfoPage_: React.FunctionComponent<{}> = () => {
     const prepReadMetadata = (data: SequenceMeta) => {
         if (!data.reads) return <></>;
         if (!Array.isArray(data.reads))
-            return renderReadsMetadata([data.reads], 1);
-        return data.reads.map((v, i) => {
-            return renderReadsMetadata(Array.isArray(v) ? v : [v], i);
-        });
+            return (
```
[the remark this answers is not in the document]
Comment (danielreti) [reply web/src/DetailedInfoPage.tsx:240]: If data.reads is not an array then data.reads.map will fail - unless there's a neater way to merge the cases?

File: web/src/DetailedInfoPage.tsx [hunk 2]
```diff
@@ -355,52 +372,61 @@ export const DetailedInfoPage_: React.FunctionComponent<{}> = () => {
...
-            const stringifiedValue = safeValue(value);
-            return (
-                <div key={`${key}-${stringifiedValue}`}>
-                    <b>{key}:</b> {stringifiedValue}
-                </div>
-            );
-        });
+                const stringifiedValue = safeValue(value) ?? <em>no-value</em>;
+                return (
+                    <div key={`${key}-${stringifiedValue}`}>
```
[the remark this answers is not in the document]
Comment (danielreti) [reply web/src/DetailedInfoPage.tsx:437]: Sorry I don't follow - were we not already using stringifieidvalue as a key before?

File: web/src/project/ProjectSummary.tsx [hunk 3]
```diff
@@ -402,6 +424,18 @@ export const ProjectSummary = () => {
         );
     };
 
+    const multiQCReports = () => {
+        return Object.entries(REPORT_TYPES).map(([key, value]) => (
```
[the remark this answers is not in the document]
Comment (danielreti) [reply web/src/project/ProjectSummary.tsx:428]: Yes mostly they won't exist but Cas wanted a quick and dirty implementation. We can clean it up later? Once it's added to the backend. As per this issue - https://github.com/populationgenomics/sample-metadata/issues/335 
He also wants a seqr button which I haven't added yet.

File: web/src/DetailedInfoPage.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (illusional) [reply web/src/DetailedInfoPage.tsx:437]: Yes, but you've added `?? <em>no-value</em> as a potential stringifiedValue

File: web/src/project/ProjectSummary.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (illusional) [reply web/src/project/ProjectSummary.tsx:428]: Perfect

File: web/src/DetailedInfoPage.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (danielreti) [reply web/src/DetailedInfoPage.tsx:437]: Oh I see - will fix!

**Pull request merged**